src/APICalls.js

```diff
@@ -1,40 +1,43 @@
 
 const fetchData = () => {
-    fetchUserData();
-    fetchHydrationData();
-    fetchSleepData();
-    fetchActivityData();
-}
-
-const fetchUserData = () => {
-    fetch("http://localhost:3001/api/v1/users")
-    .then(response => response.json())
-    .then(userData => console.log(userData))
-    // function that would change dom display
-    .catch(err => console.log(err.message))
-}
-
-const fetchHydrationData = () => {
-    fetch("http://localhost:3001/api/v1/hydration")
-    .then(response => response.json())
-    .then(hydrationData => console.log(hydrationData))
-    // function that would change dom display
-    .catch(err => console.log(err.message))
-}
-const fetchActivityData = () => {
-    fetch("http://localhost:3001/api/v1/activity")
-    .then(response => response.json())
-    .then(activityData => console.log(activityData))
-    // function that would change dom display
-    .catch(err => console.log(err.message))
-}
+    let userData =  fetch("http://localhost:3001/api/v1/users")
+        .then(response => response.json())
+        .then(userData => {
+            console.log(userData)
+            return userData
+        })
+        .catch(err => console.log(err.message))
+    
+    let hydrationData = fetch("http://localhost:3001/api/v1/hydration")
+        .then(response => response.json())
+        .then(hydrationData => {
+            return hydrationData
+        })
+        .catch(err => console.log(err.message))
```

Comment:
Make a note to add real error message for users - improve error handling on these functions...